ai_economist/foundation/components/redistribution.py

```diff
@@ -357,6 +404,9 @@ def curr_marginal_rates(self):
                 np.array(self.us_federal_single_filer_2018_scaled), self.curr_rate_max
             )
 
+        if self.tax_model == "saez":
+            return np.minimum(self.curr_bracket_tax_rates, self.curr_rate_max)
+
         if self.tax_model == "fixed-bracket-rates":
```

Comment:
elif, not if

Reply:
Since there's a return statement in the 'if', Pylint says the elif is unnecessary.

Reply:
because you're considering multiple cases of values of self.tax_models, "if elif elif else" is clearer, or use "match ...  case case"

Reply:
Used if..elif for readability